app/retail/templates/shared/analytics.html

```diff
@@ -16,6 +16,6 @@
 {% endcomment %}
 {% if not suppress_sumo and env == 'prod' %}
   {% if not user.is_authenticated %}
-  <script async>(function(s,u,m,o,j,v){j=u.createElement(m);v=u.getElementsByTagName(m)[0];j.async=1;j.src=o;j.dataset.sumoSiteId='0f081a6f95a3644ff88bf8da28b0ac85e88083ae84b6d685a8039be24680e53c';v.parentNode.insertBefore(j,v)})(window,document,'script','//load.sumo.com/');</script>
+  <script>(function(s,u,m,o,j,v){j=u.createElement(m);v=u.getElementsByTagName(m)[0];j.async=1;j.src=o;j.dataset.sumoSiteId='0f081a6f95a3644ff88bf8da28b0ac85e88083ae84b6d685a8039be24680e53c';v.parentNode.insertBefore(j,v)})(window,document,'script','//load.sumo.com/');</script>
```

Comment:
Since sumo provide this code and wait to be injected like is I should say let the async, is not going to make diff for us and will be secured to this script doesn't block the rendering